src/pages/public_registration/public_files/PendingFileInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import { Skeleton, Typography } from '@mui/material';
+import { Trans } from 'react-i18next';
+import { useFetchOrganization } from '../../../api/hooks/useFetchOrganization';
+import { useFetchPerson } from '../../../api/hooks/useFetchPerson';
+import { useFetchUserQuery } from '../../../api/hooks/useFetchUserQuery';
+import { getFullCristinName } from '../../../utils/user-helpers';
+import { PendingFilesInfo } from './PendingFilesInfo';
+
+interface PendingFileDetailsProps {
+  uploadedBy: string;
+}
+
+export const PendingFileDetails = ({ uploadedBy }: PendingFileDetailsProps) => {
+  const userQuery = useFetchUserQuery(uploadedBy);
```

Comment:
Ingen `gcTime` på `useFetchUserQuery` eller `useFetchPerson`. Kan ikke se noen god grunn til det?

Reply:
Etter litt frem og tilbake her har jeg foreløpig landet på å ikke ha langvarig caching som default, da det fort kan bite oss bak når man endrer på user/person. I stedet kan vi sette lenger cache-tid der vi selv føler det er fornuftig, som feks her når vi bare vil ha navn.

Det blir litt mer kode da, så er ikke helt frelst for denne løsningen, men tror det vil hjelpe oss å unngå bugs samtidig som vi har fleksibiliteten til å optimalisere der vi ser behov. Blir litt ekstra ansvar på utvikler å vurdere dette da